install.sh

```diff
@@ -46,6 +46,10 @@ mysql -u root -p engelsystem < db/update.sql
 echo "Edit the database name username and password in config/config.php file"
 sudo cp config/config-sample.default.php config/config.php
 
+echo "Adding folder permission for Apache"
+sudo chown -R www-data /var/www/html/engelsystem/
+sudo chmod -R g+rw /var/www/html/engelsystem/
```

Comment:
When you set the "owner" to be _www-data_, the permission for _group_ doesn't affect any more. Even if "group" is not permitted to write, Apache can still write, because user of Apache, _www-data_, matches "owner" of the folders.